lib/vehicle.rb

```diff
@@ -22,4 +26,14 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def plate_type(vehicle)
+    if vehicle.antique?
```

Comment:
This method does not take an argument in the interaction pattern:
```ruby
cruz.plate_type
```
How could you write it without an argument?